.github/workflows/scan.yaml

```diff
@@ -0,0 +1,49 @@
+# Copyright 2024 Tetrate
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Scan
+
+on:
+  push:
+    branches:
+      - main
+      - release-**
+    tags: [ 'v[0-9]+.[0-9]+.[0-9]+**' ]  # Ex. v0.2.0, v0.2.1-rc2
+  pull_request:
+    branches:
+      - main
+      - release-**
+  workflow_dispatch: {}
+
+env:
+  GOPROXY: https://proxy.golang.org
+
+jobs:
+  scan:
+    runs-on: ubuntu-latest
+    env:
+      DOCKER_TAG: scan
+      DOCKER_HUB: local
+      DOCKER_TARGETS: linux-amd64
+    steps:
+      - uses: docker/setup-qemu-action@v3
+        with:
+          platforms: amd64,arm64
```

Comment:
why do you need to `arm64` if DOCKER_TARGETS is set to `amd64` ?

Reply:
correct :)